packages/volto-blocks/src/components/blocks/IconsAndText/ViewItem.tsx

```diff
@@ -0,0 +1,91 @@
+import cx from 'classnames';
+
+import type { ArrayElement } from '@plone/types';
+import { TextBlockView } from '@plone/volto-slate/blocks/Text';
+import { UniversalLink } from '@plone/volto/components';
+import { flattenToAppURL } from '@plone/volto/helpers';
+import type { IconsAndTextData } from '@redturtle/volto-blocks/components/blocks/IconsAndText/schema';
+import styles from '@redturtle/volto-blocks/components/blocks/IconsAndText/styles.module.scss';
+
+type Props = {
+  data: ArrayElement<IconsAndTextData['columns']>;
+};
+
+export default function ViewItem({ data }: Props) {
+  const icon = data.iconImage;
+
+  return (
+    <div
+      className={cx(
+        'block-iconsandtext-column',
+        styles['column-block'],
+        styles['divider_' + data.dividerPosition],
+      )}
+    >
+      <div
+        className={cx(
+          'column-head',
+          styles['column-head'],
+          styles['header-text-' + data.headerTextPosition],
+        )}
+      >
+        {icon && (
+          <div
+            className={cx(
+              'column_icon size_',
+              styles['column_icon'],
+              styles[`size_${data.iconSize}`],
+            )}
+          >
+            <img
+              src={flattenToAppURL(icon) + '/@@images/image/teaser'}
+              alt=""
+              role="presentation"
+              aria-hidden="true"
+            />
+          </div>
+        )}
+        {data.headerTextPosition && (
+          <div className={cx(styles['header-text'])}>
+            {data.title && (
+              <h3
+                className={cx('block-iconsandtext-title title', styles.title)}
```

Comment:
```suggestion
                className={cx('block-iconsandtext-title', styles.title)}
```
`title` is way too generic for a class here